mappings/servers/fara/metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "fara",
+    "name": "FaraMC",
+    "description": "The next generation of Minecraft Minigames.",
+    "addresses": [
+        "faramc.uk"
+    ],
+    "primaryAddress": "faramc.uk",
+    "minecraftVersions": [
+        "1.8.*",
+        "1.20.*"
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#800080",
+    "secondaryColor": "#6a5acd",
+    "primaryLanguage": "en",
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "EU",
+    "regions": [
+        "EU"
+    ],
+    "gameTypes": [
+        "PvP", "Minigames"
+    ],
+    "website": "https://faramc.uk",
+    "store": "https://store.faramc.uk",
+    "compliance": {
```

Comment:
Remove this section
Our scheme does not support it

More info can be found [here](https://github.com/CheatBreakerNet/Client-API/blob/master/mappings/metadata.schema.json)